.github/workflows/deploy.yaml

```diff
@@ -13,6 +13,9 @@ jobs:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Create Tutorial Zip
+        run: zip content/docs/SUSHI/tutorial/fsh-tutorial.zip fsh-tutorial/* -r -FS
+
```

Comment:
I might be overlooking something, but if we have the zip file added in the source code, do we need to re-create the zip file on deployment? If this is to ensure it always stay up to date, do we need a version check in or can we just gitignore it?